doc/WebSite/Multi-Tenancy.md

```diff
@@ -213,9 +225,9 @@ IMustHaveTenant:
     public class Product : Entity, IMustHaveTenant
     {
         public int TenantId { get; set; }
-
+    
```

Comment:
Shall we revert these whitespace additions?

Reply:
Done. I guess my editor adds them automatically.